assets/css/src/amp-default.css

```diff
@@ -4,7 +4,9 @@
  * width such as in the post content so that the contents do not get stretched or cropped.
  * See <https://github.com/ampproject/amphtml/issues/21371#issuecomment-475443219>.
  */
-.amp-wp-enforced-sizes {
+amp-img.amp-wp-enforced-sizes,
+amp-anim.amp-wp-enforced-sizes,
+.amp-wp-unknown-size {
```

Comment:
For `amp-img` and `img` alike, having `object-fit:contain` is really only relevant when the original dimensions are unknown. Otherwise, this is currently resulting in overriding `object-fit` on any processed image. This behavior is retained for `amp-img`/`amp-anim` for now, but when native `img` are enabled we'll only apply it to images with unknown size dimensions.

Reply:
This is especially important to retain for `amp-img`/`amp-anim` specifically since these custom elements lack intrinsic object sizing in the browser, meaning that if the element has `display:block` the image could end up getting stretched.